src/dmd/link.d

```diff
@@ -152,6 +152,44 @@ version (Posix)
     }
 }
 
+version (Windows)
+{
+    private const(char)* quoteArgIfNeeded(const(char)* arg)
+    {
+        if (arg[0] == '"')
```

Comment:
The existing workaround should not break, i.e. quotes later inside the argument should be checked, too.

Reply:
You mean there might be people using stuff like `"-La.lib b.lib \"c.lib\""`? *shudder*

Reply:
No, but `"/LIBPATH:\"folder with spaces\""` if there are no other solutions.

Reply:
Ah right (I'd use `"-L\"/LIBPATH:folder with spaces\"", that alternative wouldn't have crossed my mind).

Reply:
As a side note, `dmd -L"abc def"` now works as expected, so presumably `dmd -L/LIBPATH:"folder with spaces"` too.